python/triton/runtime/jit.py

```diff
@@ -317,7 +326,6 @@ def __init__(self, fn, version=None, do_not_specialize=None):
         self.__module__ = fn.__module__
 
     @property
-    @functools.lru_cache()
```

Comment:
This change isn't related, but the `lru_cache()` seems redundant with the `self.hash` cache.